deploy

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env sh
+set -e
+
+# set context to recipeyak — this will break on machines not configured
+# for access via docker-machine (i.e. every machine but wombat)
+eval $(docker-machine env recipeyak)
+
+# set tag to argument if provided
+tag="latest"
+if [ -n "$1" ]; then
+    tag=$1
+fi
+
+echo "Deploying tag: $tag"
+
+# pull new files
+docker pull cdignam/recipemanager_nginx:${tag}
```

Comment:
I thought  we were using the `recipeyak` namespace now.

Reply:
I'll fix it.